fileio/fileio.go

```diff
@@ -195,3 +195,21 @@ func ReadFileToSingleLineString(filename string) string {
 	}
 	return catInput
 }
+
+func WriteToFileHandle(file io.Writer, rec string) {
+	var err error
+	_, err = fmt.Fprintf(file, "%s\n", rec)
+	exception.PanicOnErr(err)
+}
+
+//Write writes a file
+func Write(filename string, records []string) {
```

Comment:
I have no objections to the code, but a couple housekeeping points. 

Since this function uses the 'Easy' functions it may make sense for this to live in easyio.go instead of fileio.go. 

The function name `Write` is generally reserved for writing a slice of bytes to an open file. It may be more fitting to call this something like `EasyWriteFile` to avoid confusion.

Reply:
Happy to move it and rename it. I'm interested in using your suggestion with the helper function above and will wait to make this change until I improve that helper function.